src/bp/ui-admin/src/Pages/Account/ChatAuthResult.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { LoginContainer } from '../Layouts/LoginContainer'
+
+const ChatAuthResult = props => {
+  const error = props.location.state && props.location.state.error
+
+  if (error) {
+    return (
+      <LoginContainer title="Authentication failed">
+        <p>{error}</p>
+      </LoginContainer>
+    )
+  }
+
+  window.close()
+
+  return (
+    <LoginContainer title="Authentication">
+      <p>Successfully authenticated. You can close this window.</p>
+    </LoginContainer>
+  )
+}
```

Comment:
I'm not sure about this ... window.close will close the current window before rendering the success message right? 

You can add id in a timeout and display a message like "window will close automatically" or whatever